#! /bin/sh -e
## 002_de-translation.dpatch by  <bubulle@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Tim Dijkstra patch for proper handling of autocomments

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0

@DPATCH@

diff -Nru --exclude=CVS geneweb-4.09+cvs20040827.old/hd/lang/lexicon.txt geneweb-4.09+cvs20040827/hd/lang/lexicon.txt
--- geneweb-4.09+cvs20040827.old/hd/lang/lexicon.txt	2004-06-07 20:42:34.000000000 +0200
+++ geneweb-4.09+cvs20040827/hd/lang/lexicon.txt	2004-11-15 18:44:36.000000000 +0100
@@ -1476,7 +1476,7 @@
 ca: després de
 cs: po
 da: efter
-de: nach
+de: nachher
 en: after
 eo: post
 es: despu&eacute;s
@@ -2106,7 +2106,7 @@
 ca: abans de
 cs: pøed
 da: f&oslash;r
-de: vor
+de: vorher
 en: before
 eo: antaù
 es: antes
